src/ray/common/ray_config_def.h

```diff
@@ -420,3 +420,7 @@ RAY_CONFIG(int, publish_batch_size, 5000)
 /// The time where the subscriber connection is timed out in milliseconds.
 /// This is for the pubsub module.
 RAY_CONFIG(uint64_t, subscriber_timeout_ms, 30000)
+
+/// Feature flag to start worker process in individual container.
+/// Currently it only works on linux.
```

Comment:
It's better to check if this flag is only enabled on linux if it's only used on linux.